Source/Core/Core/PowerPC/Interpreter/Interpreter_SystemRegisters.cpp

```diff
@@ -351,12 +356,48 @@ void Interpreter::mtspr(UGeckoInstruction _inst)
 
 	// Page table base etc
 	case SPR_SDR:
-		Memory::SDRUpdated();
+		PowerPC::SDRUpdated();
 		break;
 
 	case SPR_XER:
 		SetXER(rSPR(iIndex));
 		break;
+
+	case SPR_DBAT0L:
+	case SPR_DBAT0U:
+	case SPR_DBAT1L:
+	case SPR_DBAT1U:
+	case SPR_DBAT2L:
+	case SPR_DBAT2U:
+	case SPR_DBAT3L:
+	case SPR_DBAT3U:
+	case SPR_DBAT4L:
+	case SPR_DBAT4U:
+	case SPR_DBAT5L:
+	case SPR_DBAT5U:
+	case SPR_DBAT6L:
+	case SPR_DBAT6U:
+	case SPR_DBAT7L:
+	case SPR_DBAT7U:
+		PowerPC::DBATUpdated();
```

Comment:
No break here? (Same for IBAT, altho its at the end of the switch, so it probably wouldn't matter)